core/src/main/java/com/digitalpebble/storm/crawler/bolt/ParserBolt.java

```diff
@@ -169,7 +168,7 @@ public void execute(Tuple tuple) {
         // rely on mime-type provided by server or guess?
 
         ByteArrayInputStream bais = new ByteArrayInputStream(content);
-        Metadata md = new Metadata();
+        org.apache.tika.metadata.Metadata md = new org.apache.tika.metadata.Metadata();
```

Comment:
there is nothing about it that is specific to crawling, could be use for storing any other type of metadata. The package name is probably enough to differentiate.